.gitmodules~

```diff
@@ -0,0 +1,5 @@
+[submodule "decco"]
```

Comment:
Isn't this already in sonatype?

Reply:
Arrrrgh, it's the backup file created by emacs or vi. Shouldn't be in git. Removed.